packages/react-components/src/components/styles/UnsupportedEnvironment.styles.tsx

```diff
@@ -46,7 +46,8 @@ export const linkTextStyles: ILinkStyles = {
  */
 export const containerStyles: IStackStyles = {
   root: {
-    padding: '2rem'
+    padding: '2rem',
+    width: _pxToRem(375)
```

Comment:
why does this have a fixed width? Is it responsive to narrow screens?

Reply:
just to enforce how the longers strings wrap, will update to be maxWidth